Formula/hashlink.rb

```diff
@@ -0,0 +1,25 @@
+class Hashlink < Formula
+  desc "Virtual machine for Haxe"
+  homepage "https://hashlink.haxe.org/"
+  url "https://github.com/HaxeFoundation/hashlink.git",
+    :tag =>"1.11"
```

Comment:
This needs a revision as well.

Reply:
I can change this, but it's strange that :tag is permitted at all in that case, since it could conflict with :revision.

Reply:
It wouldn't, tags can be recreated. The revision makes sure that doesn't happen without us knowing about it.